dev/tasks/tasks.yml

```diff
@@ -994,6 +994,10 @@ tasks:
       - r-lib__libarrow__bin__linux-openssl-1.0__arrow-{no_rc_r_version}\.zip
       - r-lib__libarrow__bin__linux-openssl-1.1__arrow-{no_rc_r_version}\.zip
       - r-lib__libarrow__bin__linux-openssl-3.0__arrow-{no_rc_r_version}\.zip
+      - r-lib__libarrow__bin__darwin-arm64-openssl-1.1__arrow-{no_rc_r_version}\.zip
+      - r-lib__libarrow__bin__darwin-arm64-openssl-3.0__arrow-{no_rc_r_version}\.zip
+      - r-lib__libarrow__bin__darwin-x86_64-openssl-1.1__arrow-{no_rc_r_version}\.zip
+      - r-lib__libarrow__bin__darwin-x86_64-openssl-3.0__arrow-{no_rc_r_version}\.zip
```

Comment:
They will be uploaded to `https://apache.jfrog.io/ui/native/arrow/r/14.0.0/libarrow/bin/darwin-{arm64,x86_64}-openssl-{1.1,3.0}/arrow-*.zip`. Is it expected?

https://github.com/apache/arrow/blob/50de296b0b62e54ee3a67401f277afba9b9d0433/dev/release/binary-task.rb#L1931-L1933

Reply:
Yest that should be corrcect, matching the existing linux and windows binaries

Reply:
See also: https://github.com/ursacomputing/crossbow/actions/runs/6386952098/job/17335061646#step:12:18